testcontainers-dapr/pom.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.dapr</groupId>
+        <artifactId>dapr-sdk-parent</artifactId>
+        <version>1.12.0-SNAPSHOT</version>
```

Comment:
Since it is not stable yet, make the major version as 0, like 0.12.0-SNAPSHOT - that is the convention we are using for any new artifact that is not stable in the SDK yet. It tracks the minor and patch versions but has 0 as major.

Reply:
got it..

Reply:
OK. This one remains as-is but thanks for fixing in the artifact version.